SU-SWS/stanford_news
# D8CORE-2070: changing the MS on the header for accelerants.

# READY FOR REVIEW

# Summary
- Changing the header MS spacing on the h1 for News, People, Events.

# Needed By (Date)
- Thursday

# Urgency
- medium

# Steps to Test

1. pull in this css
2. compile it
3. verify the h1 has the MS 7 for top and bottom margins.

# Affected Projects or Products
- News, Events, and Person

# Associated Issues and/or People
- D8CORE-2070
- https://github.com/SU-SWS/stanford_news/pull/88
- https://github.com/SU-SWS/stanford_person/pull/96
- https://github.com/SU-SWS/stanford_events/pull/11

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
